core/include/scipp/core/element/geometric_operations.h

```diff
@@ -16,7 +16,68 @@ namespace scipp::core {
 
 namespace element {
 
-namespace geometry {
+    constexpr auto less = overloaded{
```

Comment:
These are not geometric operations, I think they accidentially ended up in the wrong header?

Reply:
moved to logical_operations.h